docs/Asahi/asahi-installation.mdx

```diff
@@ -0,0 +1,151 @@
+
+## Requirements
+There is the requirements in order to install Asahi Linux :
+- Apple Silicon chipset like M1, M2 or M3 family (all supported devices
+[here](https://github.com/AsahiLinux/docs/wiki/Devices))
+- macOS 12.3 or later (Admin User)
+- At least 60GB of free disk space (Asahi Linux Desktop require around 15Go)
+- A fairly good internet connection (between 700Mo and 4Go of download)
+
+:::info
+Contrary to standard OS installation, Asahi Linux doesn't require any ISO
+file to USB sticks manipulation
+:::
+
+:::caution
+Do not forget to do some data backup before installing Asahi Linux
+even though the installer will not erase your macOS partition but as we
+always say : "You can never be too cautious"
+:::
+
+## Installation
+The installation of Asahi Linux is quite straight forward, so don't worry
+everything will be easy !
+
+- First copy/paste [this](https://asahilinux.org/) command into your terminal
+```bash
+curl https://alx.sh | sh
+```
+- Enter your sudo password :
+
+![curl](./img/install/curl.png)
+
+- Press enter to skip the welcome message
+- After some OS Information is displayed, enter 'r' to resize an existing
+partition to make space for a new OS
+
+![partition](./img/install/partition.png)
+
+- Now choose the partition size you want to leave for macOS :
+
+![partition-size](./img/install/partition-size.png)
+:::caution
+Choose your partition size wisely, it **can't** be changed after that step.
+:::
+
+- Press 'y' after the system freezing note (it's normal don't worry)
+- Wait for the partitioning process to end
+- Press 'enter' after partitioning is done
+- Press 'f' to install Asahi Linux into the newly allocated free space
+
+![partition-validate](./img/install/partition-validate.png)
+:::info
+If you can see the newly allocated space (yellow), then the partitioning
+process
+occurred without error.
+:::
+
+- Choose the OS you want to install by pressing the corresponding key
+- Press 'enter' in order to use all available free space for Asahi Linux
+- Enter a name for your newly created OS
+
+:::tip
+Enter a name you can remember or obvious like Asahi Fedora
+:::
+
+- Now the OS will be installed, wait for it to end
+- Enter macOS sudo password
+- If you want to improve Asahi Linux, press 'y' to report your install, if
+not then press 'n'
+- Press enter to reboot
+
+:::danger
+We will enter the only "complex" part, so know please strictly follow the
+instructions if you don't want to leave your new installation in an unbootable state
+:::
+
+- Here are the following instructions :
+
+![warning](./img/install/warning.png)
+:::note
+If you wat this part to be detailed, please open an issue [here](https://github.com/EpiMac/docs/issues)
+:::
+
+- After booting into Asahi Fedora, follow the installing process like a
+normal Linux OS installation
+
+:::tip
+Don't forget to update your system when first using it 😉
+```bash
+sudo pacman -Syyu
+```
+:::
+
+- **Good Job ! You're now a member of Asahi Linux Community !**
```

Comment:
```suggestion
**Good Job! You're now a member of Asahi Linux Community!**
```

This is not a installation step.

Reply:
Text modification and link to github repo for contributing in the next commit